gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/service/BundleMapperService.java

```diff
@@ -61,10 +65,17 @@ public Bundle mapToBundle(RCMRIN030000UK06Message xmlMessage) {
         var referralRequests = mapReferralRequests(ehrFolder, patient);
         addEntries(bundle, referralRequests);
 
+        var immunizations = mapImmunizations(ehrExtract, patient, List.of(new Encounter()));
```

Comment:
Another favour to ask, can you leave a comment here

// TODO: Insert encounter list (NIAD-1961)

For myself to link up later!